docs/short-example.ipynb

```diff
@@ -333,7 +337,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "res.choppers['WFM1'].wavelengths.data.hist(wavelength=300).plot()"
+    "res.choppers['WFM1'].wavelengths[0].data.hist(wavelength=500).plot()"
```

Comment:
I don't think the integer-indexing without labels is very intuitive. When looking at the code I was not aware that this was exposed to users. Is there a better way?

Reply:
The results are stored in a DataGroup with one entry for each pulse.
I thought it was the easiest way to do things, as I couldn't add another dimension since not every pulse will contain the same number of neutrons (length mismatch problem).

I think asking users to type `res.choppers['WFM1'].wavelengths['pulse:0']` is not great either....
Another way would be to dynamically add `.pulse*` attributes, so that one could do `res.choppers['WFM1'].wavelengths.pulse0.data`? Not sure if I like that...

Finally, I think using binned data instead of a DataGroup to store the pulses doesn't really help with the issue you raise?

Reply:
Ah, maybe you were saying we should use `wavelengths['pulse', 0]` to make it look like Scipp indexing?

Reply:
How often do users actually need to look at this and use the API?

Reply:
Not sure. I guess you are saying for now I should just remove the `getitem` and them think of something better once users actually ask for it? You can still get to the data without it.

Reply:
You mean it is possible to get all the data combined?

I think you can maybe leave it, unless it may be harmful?

Reply:
If I leave it, it makes it easy to e.g. plot a single pulse
```Py
res.choppers['WFM1'].tofs[0].plot()
```

Reply:
I put the `getitem` back in and also made it work with range slicing.